pkgs/applications/version-management/git-and-tools/radicle-cli/default.nix

```diff
@@ -0,0 +1,66 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+, pkg-config
+, cmake
+, installShellFiles
+, asciidoctor
+, DarwinTools
+, openssl
+, libusb1
+, AppKit
+, openssh
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "radicle-cli";
+  version = "0.6.0";
+
+  src = fetchFromGitHub {
+    owner = "radicle-dev";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-43dckx37qNBytKtyBS4uUpQ4cIFlQ7+i8NXewgmBABw=";
+  };
+
+  cargoSha256 = "sha256-NRHMdOmuUelUTFD6tFQsaJ30px0xNgR3WlBkuNOwFLY=";
+
+  nativeBuildInputs = [
+    pkg-config
+    cmake
+    installShellFiles
+    asciidoctor
+  ] ++ lib.optionals stdenv.hostPlatform.isDarwin [
+    DarwinTools
+  ];
+
+  buildInputs = [
+    openssl
+  ] ++ lib.optionals stdenv.hostPlatform.isDarwin [
+    libusb1
+    AppKit
+  ];
+
+  postInstall = ''
+    for f in $(find . -name '*.adoc'); do
+      mf=''${f%.*}
+      asciidoctor --doctype manpage --backend manpage $f -o $mf
+      installManPage $mf
+    done
+  '';
```

Comment:
Is this supposed to install a man page? If so, it didn't work for me. At least not in `nix-shell`

Reply:
Yes, this is supposed to install a man page, and `man rad` in a `nix-shell` works for me (without having it installed globally). I am on Nix 2.10.3 if that is relevant.

Reply:
OK, must've been because I ran it in docker. After installing it on the host it does work.